demisto_sdk/commands/common/tests/create_id_set_test.py

```diff
@@ -18,7 +18,7 @@ def test_create_id_set_flow(repo, mocker):
 
     id_set_content = repo.id_set.read_json_as_dict()
     assert not IsEqualFunctions.is_dicts_equal(id_set_content, {})
-    assert IsEqualFunctions.is_lists_equal(list(id_set_content.keys()), ID_SET_ENTITIES)
+    assert IsEqualFunctions.is_lists_equal(list(id_set_content.keys()), ID_SET_ENTITIES + ['Packs'])
```

Comment:
Why don't we add `Packs` to`ID_SET_ENTITIES`?

Reply:
The list `ID_SET_ENTITIES` is only used for checking duplicates in the ID set, after checking with @teizenman we decided that it is not necessary to add it to this list but only to the list - `CONTENT_ENTITIES`